src/main/features/ccj/views/confirmation.njk

```diff
@@ -31,9 +31,13 @@
 
       <h2 class="heading-medium">{{ t('What happens next') }}</h2>
       {% if reDeterminationRequestedAt !== undefined %}
-        <p>{{ t('A judge will decide what the defendant can afford to pay based on their financial details.') }}</p>
+         {% if reDeterminationBy === 'CLAIMANT' %}
```

Comment:
This should use an Enum rather than a string literal

Reply:
done